notary-commons/src/main/kotlin/com/d3/commons/sidechain/iroha/util/IrohaPaginationHelper.kt

```diff
@@ -97,12 +97,14 @@ class IrohaPaginationHelper(private val queryAPI: QueryAPI, private val pageSize
      * Returns the first account detail complied to a given predicate
      * @param storageAccountId - details holder account id
      * @param writerAccountId - details setter account id
+     * @param shufflePage - flag that makes this function to shuffle page result before applying any predicate
      * @param firstPredicate - predicate
      * @return the first account detail complied to [firstPredicate] or null
      */
     fun getPaginatedAccountDetailsFirst(
         storageAccountId: String,
         writerAccountId: String,
+        shufflePage: Boolean,
```

Comment:
What about setting default value to be backward-compatible?

Reply:
Ok. Good idea.